apps/modernization-ui/src/components/FormInputs/DatePickerInput.tsx

```diff
@@ -46,6 +49,15 @@ export const DatePickerInput = (props: DatePickerProps) => {
 
     const [error, setError] = useState(!(emptyDefaultValue || validDefaultValue));
 
+    const watchDateChange = useWatch({ control: props.control, name: 'dateOfBirth' });
```

Comment:
issue (blocker):  The component should not assume it's used within a form.  This also causes an error on any other page that uses a `DatePickerInput`

Reply:
Sure, Will check that.